### PR TITLE
feat: verify signature from event webhook

### PR DESCRIPTION
**Pull request by eshanholtz**
When enabling the "Signed Event Webhook Requests" feature in Mail Settings, Twilio SendGrid will generate a private and public key pair using the Elliptic Curve Digital Signature Algorithm (ECDSA). Once that is successfully enabled, all new event posts will have two new headers: X-Twilio-Email-Event-Webhook-Signature and X-Twilio-Email-Event-Webhook-Timestamp, which can be used to validate your events.

This SDK update will make it easier to verify signatures from signed event webhook requests by using the VerifySignature method. Pass in the public key, event payload, signature, and timestamp to validate. Note: You will need to convert your public key string to an elliptic public key object in order to use the VerifySignature method.

### CODE REVIEW

File: sendgrid/helpers/inbound/config.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def __init__(self, **opts):
             'path', os.path.abspath(os.path.dirname(__file__))
         )
         with open('{0}/config.yml'.format(self.path)) as stream:
-            config = yaml.load(stream)
+            config = yaml.load(stream, Loader=yaml.FullLoader)
```
Comment (childish-sambino) [inline sendgrid/helpers/inbound/config.py:19]: Why is this needed?

File: sendgrid/helpers/eventwebhook/eventwebhook_header.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class EventWebhookHeader:
+    """
+    This class lists headers that get posted to the webhook. Read the docs for
+    more details: https://sendgrid.com/docs/for-developers/tracking-events/event
+    """
+    SIGNATURE = 'X-Twilio-Email-Event-Webhook-Signature'
+    TIMESTAMP = 'X-Twilio-Email-Event-Webhook-Timestamp'
+
+    def __init__(self):
```
Comment (childish-sambino) [inline sendgrid/helpers/eventwebhook/eventwebhook_header.py:9]: Is this needed?

Comment (eshanholtz) [reply sendgrid/helpers/eventwebhook/eventwebhook_header.py:9]: Yup, got errors without it.

File: sendgrid/helpers/inbound/config.py (hunk 1 above)
Comment (eshanholtz) [reply sendgrid/helpers/inbound/config.py:19]: Deprecation of the default Loader: https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation
